lib/ansible/plugins/inventory/aws_ec2.py

```diff
@@ -134,9 +134,11 @@
     prefix: aws_region
 # Set individual variables with compose
 compose:
-  # Use the private IP address to connect to the host
+  # Use the private IP address to connect to the host and add additionnal variables as hostvars
```

Comment:
```suggestion
  # Use the private IP address to connect to the host and add additional variables as hostvars
```
Click on "commit suggestion" to accept this correction for a typo.